File: CSharp/Concentus/Structs/StereoWidthState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Concentus.Structs
{
    public class StereoWidthState
    {
        public int XX;
        public int XY;
        public int YY;
        public int smoothed_width;
        public int max_follower;

        public void Reset()
        {
            XX = 0;
            XY = 0;
            YY = 0;
            smoothed_width = 0;
            max_follower = 0;
        }
    }
}

File: go/opus/StereoWidthState.go
package opus

type StereoWidthState struct {
	XX             int
	XY             int
	YY             int
	smoothed_width int
	max_follower   int
}

func (s *StereoWidthState) Reset() {
	s.XX = 0
	s.XY = 0
	s.YY = 0
	s.smoothed_width = 0
	s.max_follower = 0
}
